lib-injection/sitecustomize.py

```diff
@@ -28,9 +30,19 @@ def _log(msg, *args, level="info"):
     This function is provided instead of built-in Python logging since we can't rely on any logger
     being configured.
     """
-    if not debug_mode and level == "debug":
-        return
-    print("%s:datadog.autoinstrumentation(pid: %d): " % (level.upper(), os.getpid()) + msg % args, file=sys.stderr)
+    if level == "error":
+        priority = syslog.LOG_ERR
+    elif level == "warning":
+        priority = syslog.LOG_WARNING
+    elif level == "debug":
+        priority = syslog.LOG_DEBUG
+    else:
+        priority = syslog.LOG_INFO
+    asctime = time.strftime("%Y-%m-%d %H:%M:%S", time.localtime())
+    msg = "[%s] [%s] datadog.autoinstrumentation(pid: %d): " % (asctime, level.upper(), os.getpid()) + msg % args
+    if debug_mode:
+        print(msg, file=sys.stderr)
+    syslog.syslog(priority, msg)
```

Comment:
?
```suggestion
    if debug_mode:
        print(msg, file=sys.stderr)
    else:
        syslog.syslog(priority, msg)
```

Reply:
Or since debug_mode doesn't use priority we can just move the first case to the top of the function and return early

Reply:
it wouldn't be an either or. We'd still want the system logs in this case AND the output to stderr when the debug env var is set.